posthog/src/main/java/com/posthog/java/PostHog.java

```diff
@@ -128,11 +128,11 @@ public void identify(String distinctId, Map<String, Object> properties) {
 
     /**
      * 
-     * @param distinctId distinct ID to merge. Must not be null or empty. Note: If
-     *                   there is a conflict, the properties of this person will
-     *                   take precedence.
-     * @param alias      distinct ID to merge. Must not be null or empty. Note: If
-     *                   there is a conflict, the properties of this person will be
+     * @param distinctId distinct ID of the user. Must not be null or empty. Note: If
+     *                   both the distinct ID and the lias have properties, the properties 
+     *                   of this person will take precedence.
+     * @param alias      alias ID that can be used for future identifies. Must not be null
```

Comment:
again lets be clear here that this needs to be an anonymous ID / session ID / something not previously identified